t/10-tests_overview.t

```diff
@@ -166,9 +170,30 @@ my $failing_module = $t->app->db->resultset('JobModules')->create(
 
 $get = $t->get_ok('/tests/overview' => form => {distri => 'opensuse', version => 'Factory', build => '0048', todo => 1})
   ->status_is(200);
-like(get_summary, qr/Passed: 0 Soft Failure: 1 Failed: 1/i, 'todo=1 shows all unlabeled failed');
-$t->element_exists('#res-99939', 'softfailed without failing module present');
-$t->element_exists_not('#res-99936', 'softfailed with failing module filtered out');
+like(
+    get_summary,
+    qr/Passed: 0 Soft Failure: 1 Failed: 1/i,
+    'todo=1 shows all unlabeled failed and unlabeled softfailed with failing modules'
+);
+$t->element_exists_not('#res-99939', 'softfailed without failing module filtered out');
+$t->element_exists('#res-99936', 'softfailed with unreviewed failing module present');
+
+my $review_comment = $t->app->db->resultset('Comments')->create(
+    {
+        job_id  => 99936,
+        text    => 'bsc#1234',
+        user_id => 99903,
+    });
+$get = $t->get_ok('/tests/overview' => form => {distri => 'opensuse', version => 'Factory', build => '0048', todo => 1})
+  ->status_is(200);
+like(
+    get_summary,
+    qr/Passed: 0 Failed: 1/i,
+    'todo=1 shows all unlabeled failed and unlabeled softfailed with failing modules'
```

Comment:
better update the comment to something like 'todo=1 shows only unlabeled left failed as softfailed with failing modules was labeled'